src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs

```diff
@@ -136,6 +145,138 @@ public RestoreCommand(IApplicationEnvironment env)
+                }
+                RestorePackageVersion = package.Version.ToString();
+            }
+
+            Reports.Information.WriteLine("Installing package {0} {1}", RestorePackageId.Bold(), RestorePackageVersion.Bold());
+
+            var graphNode = await restoreOperations.CreateGraphNode(
+                restoreContext,
+                new Library()
+                {
+                    Name = RestorePackageId,
+                    Version = SemanticVersion.Parse(RestorePackageVersion)
+                }, 
+                _ => true);
+
+            Reports.Information.WriteLine(string.Format("{0}, {1}ms elapsed", "Resolving complete".Green(), sw.ElapsedMilliseconds));
+
+            var installItems = new List<GraphItem>();
+            var missingItems = new HashSet<LibraryRange>();
+
+            ForEach(new GraphNode[] { graphNode }, node =>
+            {
+                if (node == null || node.LibraryRange == null)
+                {
+                    return;
+                }
+                if (node.Item == null || node.Item.Match == null)
+                {
+                    if (!node.LibraryRange.IsGacOrFrameworkReference &&
+                         node.LibraryRange.VersionRange != null &&
+                         missingItems.Add(node.LibraryRange))
+                    {
+                        Reports.Error.WriteLine(string.Format("Unable to locate {0} >= {1}", node.LibraryRange.Name.Red().Bold(), node.LibraryRange.VersionRange));
+                        success = false;
+                    }
+                    return;
+                }
+                // "kpm restore" is case-sensitive
+                if (!string.Equals(node.Item.Match.Library.Name, node.LibraryRange.Name, StringComparison.Ordinal))
+                {
+                    if (missingItems.Add(node.LibraryRange))
+                    {
+                        Reports.Error.WriteLine(
+                            "Unable to locate {0} >= {1}. Do you mean {2}?",
+                            node.LibraryRange.Name.Red().Bold(),
+                            node.LibraryRange.VersionRange, 
+                            node.Item.Match.Library.Name.Bold());
+                        success = false;
+                    }
+                    return;
+                }
+                var isRemote = remoteProviders.Contains(node.Item.Match.Provider);
+                var isAdded = installItems.Any(item => item.Match.Library == node.Item.Match.Library);
+                if (!isAdded && isRemote)
+                {
+                    installItems.Add(node.Item);
+                }
+            });
+
+            await InstallPackages(installItems, packagesDirectory, packageFilter: (library, nupkgSHA) => true);
+
+            Reports.Information.WriteLine("{0}, {1}ms elapsed", "Install complete".Green().Bold(), sw.ElapsedMilliseconds);
+
+            PrintDependencyGraph(graphNode, restoreContext.FrameworkName);
```

Comment:
Not needed anymore. We just removed this: https://github.com/aspnet/XRE/commit/65473b0a6199d9c886f358d9057cc46c35255d82